site/en/docs/crux/dashboard/index.md

```diff
@@ -41,32 +41,37 @@ The CrUX Dashboard is a [Looker Studio](https://cloud.google.com/looker-studio)
 
 ## Accessing the CrUX Dashboard
 
-To use the existing CrUX Dashboard you need to lauch the dashboard, passing the origin as a query param. To make this easier–since the origin needs to be URL encoded—you can use the below launcher, which takes an origin or URL as an input and constructs the dashboard URL for that origin:
+To use the existing CrUX Dashboard you need to lauch the dashboard, passing the origin as a query param. To make this easier–since the origin needs to be URL encoded—you can use the below launcher, which takes an origin or URL as an input and launches the appropriate dashboard URL for that origin:
 
 <style>
   form {
     display: flex;
     flex-direction: row;
     justify-content: center;
     align-items: center;
+    width: 100%;
+    max-width: 800px;
   }
   #origin {
     width: 100%;
-    max-width: 800px;
     font-size: 2em;
     line-height: 1.3em;
     text-align: center;
   }
   #origin[aria-invalid=true] {
     outline: red dashed 2px;
   }
+  #origin-submit {
+    line-height: 2.25em;
+  }
 </style>
 
 <form id="form">
   <input id="origin" aria-label="Origin" type="text" placeholder="https://developer.chrome.com" required>
```

Comment:
This input needs a `<label>` element per accessibility guidelines.

Reply:
Same comment as above, answered in https://github.com/GoogleChrome/developer.chrome.com/pull/5732#discussion_r1144912418

Reply:
Ah, I see in the W3C accessibility guidelines that an `aria-label` is an acceptable alternative. I'll approve then. Sorry!